### ISSUE TITLE
Feature Request - proctoring

### ISSUE THREAD

**Issue by worldfellow** [tecnickcom/tcexam#360]
Is there any way that we can avoid cheating of exams like implement any proctoring tool, AI etc , or simply detect if the user has navigated to any other tab or minimized the browser window for example look at this link http://serkanyersen.github.io/ifvisible.js/demo.html (the library https://github.com/serkanyersen/ifvisible.js)

**Comment by mamans86** [comment]
yes, it's quite possible to apply to TCExam. as an alternative you can use a web browser specifically designed for the exam, like the safe browser exam (SEB).

**Comment by nicolaasuni** [comment]
Thanks — proctoring (webcam/lockdown/monitoring) is a substantial feature and a good candidate for Discussions, where requirements and possible integrations can be gathered. Closing the issue.
